Headlines/src/Categories/UIImageEffects.swift

```diff
@@ -119,15 +119,15 @@ public extension UIImage {
             var red: CGFloat = 0
             var green: CGFloat = 0
             var blue: CGFloat = 0
-            
+
             if tintColor.getRed(&red, green: &green, blue: &blue, alpha: nil) {
                 effectColor = UIColor(red: red, green: green, blue: blue, alpha: effectColorAlpha)
             }
         }
-        
+
         return applyBlurWithRadius(10, tintColor: effectColor, saturationDeltaFactor: -1.0, maskImage: nil)
     }
-    
+
     public func applyBlurWithRadius(_ blurRadius: CGFloat,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/adb6600e04f8d78e85d3ed821c2844be0fbd4321/Headlines/src/Categories/UIImageEffects.swift#L131"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 125 lines<br />
<code>function_body_length</code> <code>UIImageEffects.swift:131</code></td>
    </tr>
  </tbody>
</table>